EventFilter/TotemRawToDigi/interface/VFATFrame.h

```diff
@@ -0,0 +1,150 @@
+/**********************************************************
+*
+* This is a part of the TOTEM offline software.
+* Authors:
+*    Jan Kaspar (jan.kaspar@gmail.com)
+*
+**********************************************************/
+
+#ifndef EventFilter_TotemRawToDigi_VFATFrame
+#define EventFilter_TotemRawToDigi_VFATFrame
+
+#include <vector>
+#include <cstddef>
+
+/**
+ * Representation of VFAT frame plus extra info added by DAQ.
+**/
+class VFATFrame
+{
+  public:
+    typedef unsigned short word;
```

Comment:
int16_t may be more safe overall (here and in other places where are using this word type).
from the "book"
``` short - target type will be optimized for space and will have width of at least 16 bits ```